Data/core_rulebook/cr__saves.lua

```diff
@@ -0,0 +1,35 @@
+-- Converted From LST file data\pathfinder\paizo\roleplaying_game\core_rulebook\cr__saves.lst
+-- From repository https://github.com/pcgen/pcgen at commit 11ceb52482855f2e5f0f6c108c3dc665b12af237
+DefineSave({
```

Comment:
Heavens! A lst file that appears to contain something actually useful!  Though we could probably just... hard code this. :-p

Reply:
I know, I was surprised when I saw a file with actual content in it.